polaris.shopify.com/content/version-guides/migrating-from-v11-to-v12.mdx

```diff
@@ -2895,3 +2895,8 @@ The following component's children cannot be above the bevel's `z-index` elevati
 
 Custom elements that were styled to look like the previous Polaris design language will need to be updated.
 Take the opportunity to put custom styles and components on mainline Polaris using our [components](/components) and [tokens](/tokens/color).
+
+### `.Polaris-Summer-Editions-2023` class
+
+The `<html>` element no longer receives the `.Polaris-Summer-Editions-2023` class.
+If your styles rely on this class as part of a CSS selector, you can safely remove it.
```

Comment:
Should this guidance be in the v12 to v13 migration guide instead?

Reply:
The class shouldn't have been doing anything for folks on version 12... If it was that isn't great lol and we should mention 12.16.